storage/sealer/storiface/storage.go

```diff
@@ -119,3 +122,34 @@ type Prover interface {
 
 	AggregateSealProofs(aggregateInfo proof.AggregateSealVerifyProofAndInfos, proofs [][]byte) ([]byte, error)
 }
+
+type SectorData struct {
```

Comment:
I think this would be better named `SectorLocation` or something similar, as it doesn't actually contain any sector data.